### ISSUE TITLE
Use imperative mood for one-line docstring summaries

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#1473]
This a low priority issue for better following the wider Python community style for our docstring documentation. This is a spin out from issue #1170.

For background, read PEP257 first https://www.python.org/dev/peps/pep-0257/ (and note we use reStructuredText for the docstrings, not just plain text).

To test this locally, first install ``pydocstyle``,

```
$ pip install pydocstyle
```

Then:

```
$ pydocstyle Bio/ --select D401 --count
Bio/_utils.py:38 in public function `read_forward`:
        D401: First line should be in imperative mood ('Read', not 'Reads')
...
Bio/Wise/__init__.py:26 in private function `_build_align_cmdline`:
        D401: First line should be in imperative mood; try rephrasing (found 'Helper')
416
```

If we do manage to make ``pydocstyle`` happy, then ``D401`` can be enforced on TravisCI/Tox by updating file ``Bio/.flake8`` which currently ignores it.

Given there are hundreds of docstrings to review, it makes sense to do this in batches, for example by top level folder. If you are going to work on this please comment here first to avoid overlap of effort (e.g. "I'm going to work on ``Bio/BLAST/`` next).

**Comment by MarkusPiotrowski** [comment]
I'm going to work on ```Bio/SeqUtils```

**Comment by MarkusPiotrowski** [comment]
I go on with ```Alphabet```, ```Compass``` and ```Data```

**Comment by Gasta88** [comment]
I'll take `Bio/_utils.py`, `Bio/SeqRecord.py` and `Bio/phenotype`.

**Comment by Gasta88** [comment]
I'll start with `Bio/Nexus`, `Bio/Wise` and `Bio/PopGene`

**Comment by Gasta88** [comment]
I'll work on `Bio/Entrez`, `Bio/PDB`, `Bio/FSSP` and `Bio/Phylo`.

**Comment by peterjc** [comment]
Wow - this is going quicker than I expected - thanks for all the work so far!

**Comment by Gasta88** [comment]
Working on `Bio/Uniprot`, `Bio/motifs`, `Bio/KEGG`, `Bio/NMR` and `Bio/SCOP`.

**Comment by Gasta88** [comment]
I'll take `Bio/SeqIO`, `Bio/Crystal` and `Bio/SearchIO`.

**Comment by peterjc** [comment]
I'll do ``Bio/Sequencing/*.py`` now...

**Comment by Gasta88** [comment]
I'll do `Bio/Pathway`, `Bio/NeuralNetwork` and `Bio/Blast`.

**Comment by peterjc** [comment]
Good work everyone. We're almost there - just three left:

```
$ pydocstyle Bio/ --select D401 --count
Bio/Data/CodonTable.py:71 in public method `__str__`:
        D401: First line should be in imperative mood ('Return', not 'Returns')
Bio/Data/CodonTable.py:219 in public function `list_ambiguous_codons`:
        D401: First line should be in imperative mood ('Extend', not 'Extends')
Bio/Data/CodonTable.py:396 in public function `register_ncbi_table`:
        D401: First line should be in imperative mood ('Turn', not 'Turns')
3
```

Markus' pull request #1501 will fix these (and adds the latest NCBI genetic codes to our codon tables).

Once that's merged, we can start enforcing ``D401`` in the ``Bio/.flake8`` configuration.

**Comment by peterjc** [comment]
Closed by #1542, thank you for everyone else for all the docstring changes
